### PR TITLE
Kill scriptaculous

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

**Comment by apmasell** [comment]
I've fixed the `setBaseUrl` and `doOnSuccess` problem.
The used pools stuff doesn't seem to actually use Fluxion, so I'm not sure what's going on there.
Making the list pages slower is outside the scope of this PR.

**Comment by apmasell** [comment]
The ready to run page is legitimately empty on the prod DB. If you add a pool to a run, it shows up there.

**Comment by callunity** [comment]
Huh, that's interesting about the ready to run page. I wonder when that broke.

File: miso-web/src/main/webapp/pages/external/externalHeader.jsp [hunk 1]
```diff
@@ -118,7 +118,6 @@
 </table>
 
 <script type="text/javascript">
-    setBaseUrl("<c:url value="/miso/external/"/>");
```
Comment (callunity) [inline miso-web/src/main/webapp/pages/external/externalHeader.jsp]: This is important! Keep this one!

Comment (callunity) [inline miso-web/src/main/webapp/pages/external/externalHeader.jsp]: Or rather, this does need to be distinct from the base MISO URL.

Comment (callunity) [inline miso-web/src/main/webapp/pages/external/externalHeader.jsp]: (That being said, the login page external webapp doesn't seem to be complaining about this line being deleted so I will approve it)

Comment (apmasell) [inline miso-web/src/main/webapp/pages/external/externalHeader.jsp]: Looking at the deleted code, this only matters for the `activityInput.js` URLs, which we never used.

**Pull request merged**